clients/admin-ui/src/features/data-discovery-and-detection/DiscoveryMonitorBreadcrumbs.tsx

```diff
@@ -1,91 +1,62 @@
-import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from "fidesui";
-import { useRouter } from "next/router";
+import { Icons } from "fidesui";
 
-import { DatabaseIcon } from "~/features/common/Icon/database/DatabaseIcon";
-import { DatasetIcon } from "~/features/common/Icon/database/DatasetIcon";
-import { FieldIcon } from "~/features/common/Icon/database/FieldIcon";
-import { TableIcon } from "~/features/common/Icon/database/TableIcon";
+import {
+  NextBreadcrumb,
+  NextBreadcrumbProps,
+} from "../common/nav/v2/NextBreadcrumb";
 
 interface DiscoveryMonitorBreadcrumbsProps {
   resourceUrn?: string;
   parentLink: string;
   onPathClick?: (urn: string) => void;
 }
 
-const MONITOR_BREADCRUMB_ICONS = [
-  <DatabaseIcon key="database" boxSize={4} />,
-  <DatasetIcon key="dataset" boxSize={5} />,
-  <TableIcon key="table" boxSize={5} />,
-  <FieldIcon key="field" boxSize={5} />,
+export const DATA_BREADCRUMB_ICONS = [
+  <Icons.Layers key="layers" />,
+  <Icons.Db2Database key="dataset" />,
+  <Icons.Table key="table" />,
```

Comment:
The new icons look great. Although I did find the Table one to be confusing, because it looks like the Fields one we have right now. Can we consider maybe using the <TableSplit /> instead?

Reply:
We'll be keeping the Carbon Table icon.